media/css/firefox/new/_other-platforms.scss

```diff
@@ -65,15 +65,33 @@ html[dir="rtl"] #other-platforms-modal-link {
         }
     }
 
+    .download-platform-lists {
+        margin: 20px 40px;
+    }
+
     .download-platform-list {
         list-style: none;
         margin: 20px auto;
         text-align: center;
 
+        &.recommended li {
+            font-weight: bold;
+        }
+
+        &.traditional {
+            border-width: 1px 0;
+            border-color: #D7D7DB;
+            border-style: solid;
+
+            li {
+                opacity: .7;
```

Comment:
With the opacity set here this text fails a11y contrast checks. I'd suggest relying on just the bold font weight to highlight recommended vs traditional.

Reply:
@kyoshino could you update your patch please? Thanks

Reply:
Okay, removed the `opacity` property.